web/js/naturalEvents/ui.js

```diff
@@ -251,12 +259,11 @@ wv.naturalEvents.ui = wv.naturalEvents.ui || function (models, ui, config, reque
 
     if (event.geometries.length > 1) {
       var eventIndex = 0;
-      _.each(event.geometries, function (geometry) {
-        eventIndex++;
+      lodashEach(event.geometries, function (geometry) {
+        eventIndex = eventIndex + 1;
```

Comment:
Keep looking at this line wondering why this changed?

Reply:
The linter didn't like `eventIndex++`, but `eventIndex = eventIndex + 1` is the same thing, and the linter doesn't complain.